apisix/plugins/limit-count/limit-count-redis.lua

```diff
@@ -44,7 +44,12 @@ local function redis_cli(conf)
 
     red:set_timeouts(timeout, timeout, timeout)
 
-    local ok, err = red:connect(conf.redis_host, conf.redis_port or 6379)
+    local sock_opts = {
+        ssl = conf.redis_ssl,
+        ssl_verify = conf.redis_ssl_verify
+    }
+
+    local ok, err = red:connect(conf.redis_host, conf.redis_port or 6379, sock_opts)
```

Comment:
```suggestion
    local ok, err = red:connect(conf.redis_host, conf.redis_port, sock_opts)
```
It seems that we don't need a default value because it is already defined in the schema.